website/static/js/comment.js

```diff
@@ -77,19 +261,50 @@ var BaseComment = function() {
 
     self.replying = ko.observable(false);
     self.replyContent = ko.observable('');
+    self.replyMentions = ko.observableArray([]);
+
+    self.saveContent = ko.computed(function() {
+        var content = self.replyContent();
+        if (!content) {
+            content = '';
+        }
+        self.replyMentions([]);
+        var regex = /<span.*?data-atwho-guid="([a-z\d]{5})".*?>((@|\+)[\w\s]+)<\/span>/;
+        var matches = content.match(/<span.*?data-atwho-guid="([a-z\d]{5})".*?>((@|\+)[\w\s]+)<\/span>/g);
+        if (matches) {
+            for (var i = 0; i < matches.length; i++) {
+                var match = regex.exec(matches[i]);
+                var guid = match[1];
+                var mention = match[2];
+                var url = '/' + guid + '/';
+                content = content.replace(match[0], '['+ mention + '](' + url + ')');
+
+                if (guid && self.replyMentions.indexOf(guid) === -1) {
+                    self.replyMentions.push(guid);
+                }
+            }
+        }
+        return content.replace(/<br>/g, '&#13;&#10;');
```

Comment:
> Comments are rendered as markdown which requires two returns to display as a line break. I believe this is normal markdown behavior.

Ah, right, of course. Sorry. Line breaks on newline are part of GitHub-flavored markdown. Now that I look, I don't find it mentioned in their [docs](https://help.github.com/categories/writing-on-github/), but nonetheless:

foo
bar

;-)

I've confirmed that line breaking works as expected in production. However, it appears to be broken on this PR.

# Production

![working in production](https://cloud.githubusercontent.com/assets/134455/15874208/1cae35ac-2cd0-11e6-9295-816d74d96117.gif)

# Here

![broken on this PR](https://cloud.githubusercontent.com/assets/134455/15874261/59a16bdc-2cd0-11e6-991a-f05dbcf2bbc1.gif)